inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -114,6 +124,9 @@
     @org.springframework.beans.factory.annotation.Value(value = "${data.path}/kb")
     private File dataDir;
 
+    @org.springframework.beans.factory.annotation.Value(value = "${repository.path}/luceneIndex")
```

Comment:
The KB lucene indexes should be stored next to the KB stores, i.e. under `${repository.path}/kb`.